docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,111 @@
+# Provider Networks
+
+
+## Table of Contents
+- [Provider Networks](#provider-networks)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Common prerequisites](#common-prerequisites)
+  - [Considerations when creating provider networks](#considerations-when-creating-provider-networks)
+  - [Deploying cluster with primary interface on a provider network with IPI](#deploying-cluster-with-primary-interface-on-a-provider-network-with-ipi)
+  - [Known issues](#known-issues)
+
+
+## Introduction
+
+Provider networks map directly to an existing physical network in the data center.
+Useful network types in this category include flat (untagged), VLAN (802.1Q tagged) and VXLAN. 
+OpenShift clusters can be deployed on provider networks, so the installer won't need to create
+tenant networks nor floating IPs.
+More information can be found about provider networks terminology [here][1].
+
+
+## Common prerequisites
+
+* [Neutron service][2] is enabled and accessible through the [OpenStack Networking API][3].
+* Neutron should be configured with the [port_security][4] extension so the installer can
```

Comment:
The neutron doc suggests we need the `port-security` and `allowed-address-pairs` extensions.

Reply:
ack, I'll add it.